<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>schoolLibrarian</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>book-3-fill.svg</normaloff>book-3-fill.svg</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Suchbegriff:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>SUCHEN</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>search-fill.svg</normaloff>search-fill.svg</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(188, 220, 244);
selection-color: rgb(0, 0, 0);</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Filtern nach:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_Field">
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="comboBox_Existing"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBox_Location"/>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_lastEdited">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(225, 225, 225);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>durch:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>zuletzt bearbeitet:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="lineEdit_by">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(225, 225, 225);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_created">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(225, 225, 225);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>erstellt:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionEdit_Entry"/>
   <addaction name="actionDelete_Entry"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>add-circle-fill.svg</normaloff>add-circle-fill.svg</iconset>
   </property>
   <property name="text">
    <string>Neuer Eintrag</string>
   </property>
  </action>
  <action name="actionEdit_Entry">
   <property name="icon">
    <iconset>
     <normaloff>pencil-fill.svg</normaloff>pencil-fill.svg</iconset>
   </property>
   <property name="text">
    <string>Eintrag Ã¤ndern</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>settings-5-fill.svg</normaloff>settings-5-fill.svg</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="icon">
    <iconset>
     <normaloff>delete-bin-2-fill.svg</normaloff>delete-bin-2-fill.svg</iconset>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
   <property name="toolTip">
    <string>Delete current book entry</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>information-fill.svg</normaloff>information-fill.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>comboBox_Field</tabstop>
  <tabstop>comboBox_Location</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
